README.rst

```diff
@@ -1,7 +1,26 @@
+flit
```

Comment:
The README is included as the front page of the docs, so please leave off this title. I prefer READMEs to open with text anyway.

We'll probably need to remove the logo from the docs so that it's not there twice, as well.